packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -1050,26 +1062,31 @@ operations will be generated by the CLI.`,
    * @param ctx The transformer context.
    * @param resolverResourceId The logical id of the get resolver.
    * @param rules The auth rules to apply.
+   * @param parent ObjectDefinition of the type
+   * @param modelConfiguration auth configuration for the model
+   * @param updateSchema flag to indicate if protection needs to update the schema or just the resolver. When a connection is being protected
+   * the schema should not be updated
    */
   private protectGetQuery(
     ctx: TransformerContext,
     resolverResourceId: string,
     rules: AuthRule[],
     parent: ObjectTypeDefinitionNode | null,
     modelConfiguration: ModelDirectiveConfiguration,
+    updateSchema: boolean,
```

Comment:
Is there a reason we only need this on the GET query?

Reply:
We use the same thing for connected models and that causes un desired update to schema.